app-network/ucx/autobuild/defines

```diff
@@ -0,0 +1,24 @@
+PKGNAME=ucx
+PKGSEC=science
+PKGDEP="gcc glibc libtool pkg-config rdma-core fuse-3"
```

Comment:
Why is `pkg-config` listed as a dependency? Same goes for `libtool`, does it depend on `libltdl.so.7`?

Reply:
No. `pkg-config` was used to deal with path of dynamic libraries. But since @jiegec modified the logic, it is no longer needed.